docs/developer/guidelines/dashboards.md

```diff
@@ -54,128 +49,115 @@ Ensure logos render correctly on the Dashboard List page and within the preset d
 
-### Logos
+![Elasticsearch dashboard](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/example_dashboard.png)
 
-Integration logos should be placed at the top-left of the dashboard and centered on the [image widget](https://docs.datadoghq.com/dashboards/widgets/image/).
+2. Use Group widgets to title and group sections, rather than note widgets as you might on a screenboard. Use partial width groups to display groups side-by-side. Most dashboards should display every widget within a group.
 
-The suggested height for logos is around 12 grid units.
+![Group titles](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/group_titles.png)
 
-### Labels
+3. Timeseries widgets should be at least 4 columns wide in order not to appear squashed on smaller displays
 
-Labels are created from the [Notes & Links](https://docs.datadoghq.com/dashboards/widgets/note/) widget type.
+![Squished timeseries](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/squished_timeseries.png)
 
-#### Metric Categories
-Labels are used to divide the screenboard in visually comprehensive sections.
+4. Stream widgets should be at least 6 columns wide for readability (half the dashboard width.) Avoid placing full-width stream widgets in the middle of a dashboard as they’ll “trap” scroll events. It’s also useful to put stream widgets in a group by themselves so they can be collapsed. Add an event stream only if the service monitored by the dashboard is reporting events. Use `sources:service_name`.
 
-- Category labels have `vivid blue` backgrounds to contrast with the board and widgets.
-- Text size is heading level 1 `#` and horizontally centered.
+![Stream widgets](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/stream_widgets.png)
 
-    !!! important
-        Do not use bold or all capitalized letters because it can make the dashboard harder to read.
-         _See the labels `Job Summary` and `Page Summary` in the Ignite dashboard example._
+5. Which widgets best represent your data? Try using a mix of widget types and sizes. Explore visualizations and formatting options until you’re confident your dashboard is as clear as it can be. Sometimes a whole dashboard of timeseries is ok, but other times variety can improve things. There are different types of metric widgets. The most commonly used are [timeseries](https://docs.datadoghq.com/dashboards/widgets/timeseries/), [query values](https://docs.datadoghq.com/dashboards/widgets/query_value/), and [tables](https://docs.datadoghq.com/dashboards/widgets/table/). For more information on the available widget types, see the [list of supported dashboard widgets](https://docs.datadoghq.com/dashboards/widgets/).
```

Comment:
```suggestion
5. Which widgets best represent your data? Try using a mix of widget types and sizes. Explore visualizations and formatting options until you’re confident your dashboard is as clear as it can be. Sometimes a whole dashboard of timeseries is ok, but other times variety can improve things. The most commonly used metric widgets are [timeseries](https://docs.datadoghq.com/dashboards/widgets/timeseries/), [query values](https://docs.datadoghq.com/dashboards/widgets/query_value/), and [tables](https://docs.datadoghq.com/dashboards/widgets/table/). For more information on the available widget types, see the [list of supported dashboard widgets](https://docs.datadoghq.com/dashboards/widgets/).
```